mne/viz/misc.py

```diff
@@ -110,6 +110,8 @@ def plot_cov(cov, info, exclude=[], colorbar=True, proj=False, show_svd=True,
 
     fig_cov, axes = plt.subplots(1, len(idx_names),
                                  figsize=(2.5 * len(idx_names), 2.7))
+    if isinstance(axes, plt.Axes):
+        axes = [axes]
```

Comment:
a clearer way is to do `axes = plt.subplots(..., squeeze=False)[0]`, then you are guaranteed it is a 1D array with 1 or more elements.